README.md

```diff
@@ -35,5 +35,12 @@ A Persian (Farsi) Font
            url('Samim-Bold.ttf') format('truetype');
       font-weight: bold;
     }
+    
+    body {
+        /* optional enhancements for better rendering */
+        text-rendering: auto;
+        -webkit-font-smoothing: antialiased;
+        -moz-osx-font-smoothing: grayscale;
```

Comment:
True, I'm aware of that and that's why I wrote "Optional". 
Since "Font Awesome" has also included these lines from the very first version and apparently they haven't had any problem with it, I thought it could be a nice option to include it here and let the user decide. What do yo think?